portfolio/src/main/webapp/canada.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Photo Blog</title>
+	<link href='https://fonts.googleapis.com/css?family=Raleway:400,800' rel='stylesheet' type='text/css'>
+	<link rel="stylesheet" type="text/css" href="photos.css">
+</head>
+<body>
+
+<p>Canada/2015,2016</p>
+
+<div class="row"> 
+  <div class="column">
```

Comment:
Make sure tabs and spaces don't co-exist in a project, different editors interpret tabs in a different way

Checkout superficially https://stackoverflow.com/questions/12614132/why-do-tabs-line-up-differently-in-different-text-editors